docs/_docs/app-checks-reference.md

```diff
@@ -0,0 +1,655 @@
+---
+title: Automated App Development Checks
+order: 18
+layout: post-toc
+redirect_from: /docs/
+---
+
+# Automated App Development Checks
+
+In accordance with our [app development guide](https://zapier.com/developer/documentation/v2/app-dev-guide/), we run your app through a set of automated checks to ensure it's giving our users the best possible experience. Violations come in two flavors:
```

Comment:
This links to the old doc site. Maybe we can just remove the link here?

Reply:
good catch!

Reply:
ah, we've actually got that in the new docs as well: https://platform.zapier.com/partners/planning-guide